test/core/excel/WriteEmptyListToExcel.DYN

```diff
@@ -0,0 +1,23 @@
+<Workspace Version="0.8.1.1403" X="17" Y="66" zoom="1" Name="Home" RunType="Manual" RunPeriod="100" HasRunWithoutCrash="True">
+  <NamespaceResolutionMap />
+  <Elements>
+    <Dynamo.Nodes.DSFunction guid="9bce1a0e-8623-4360-b48b-c0db96f47c0b" type="Dynamo.Nodes.DSFunction" nickname="Excel.WriteToFile" x="635.5" y="124.5" isVisible="true" isUpstreamVisible="true" lacing="Shortest" assembly="DSOffice.dll" function="DSOffice.Excel.WriteToFile@string,string,int,int,var[][]" />
+    <Dynamo.Nodes.CodeBlockNodeModel guid="833e45ce-a9a9-4f5c-9304-cc99053895ee" type="Dynamo.Nodes.CodeBlockNodeModel" nickname="Code Block" x="322" y="98" isVisible="true" isUpstreamVisible="true" lacing="Disabled" CodeText="&quot;test&quot;;" ShouldFocus="false" />
+    <Dynamo.Nodes.CodeBlockNodeModel guid="b0fd70fd-c4e0-42a1-9f6b-08aea7e0d2e9" type="Dynamo.Nodes.CodeBlockNodeModel" nickname="Code Block" x="413" y="183" isVisible="true" isUpstreamVisible="true" lacing="Disabled" CodeText="0;" ShouldFocus="false" />
+    <Dynamo.Nodes.CodeBlockNodeModel guid="f8dffa86-e0d1-4569-9220-f718a6c65c82" type="Dynamo.Nodes.CodeBlockNodeModel" nickname="Code Block" x="239" y="334" isVisible="true" isUpstreamVisible="true" lacing="Disabled" CodeText="{};" ShouldFocus="false" />
+    <Dynamo.Nodes.StringInput guid="a05edec7-9df2-4ac1-a133-5069e72c3257" type="Dynamo.Nodes.StringInput" nickname="String" x="508" y="32.5" isVisible="true" isUpstreamVisible="true" lacing="Disabled">
+      <System.String>
+      </System.String>
+      <System.String value="" />
```

Comment:
It seems something is wrong with StringInput because I am seeing two String nodes here.